app/views/books/index.html.erb

```diff
@@ -4,14 +4,34 @@
       <% @books.each do |book| %>
         <tr class="<%= cycle('bg-violet-50', 'bg-white') %> shadow">
           <td class="px-2 py-3">
-            <%= image_tag(book.cover_url, class: 'w-40') %>
+            <%= image_tag(book.cover_url, class: 'w-60') %>
           </td>
           <td>
-            <h1 class="text-xl font-bold mb-3 hover:cursor-pointer hover:underline"><%= book.title %></h1>
+            <%= link_to (book_path(book)) do %>
+              <h1 class="text-xl font-bold mb-3 hover:cursor-pointer hover:underline"><%= book.title %></h1>
+            <% end %>
             <p class="max-w-xl"><%= truncate(book.description, length: 200) %></p>
           </td>
+          <td>
+            <ul>
+              <li>
+                <%= link_to('Edit', edit_book_path(book),
+                  class: 'hover:underline') %>
```

Comment:
**[Minor]** The indentation here looks a bit weird. 😅  Just out of curiosity, are you using some sort of formatter for the ERB files? Asking because the `link_to` below looks aligned on the function arguments, but this one looks very different.